terraform/modules/tf_stream_alert_cloudtrail/main.tf

```diff
@@ -1,3 +1,87 @@
+// KMS key for encrypting CloudTrail logs
```

Comment:
Is there a reference in AWS docs for the policies below? If so could you add it? 🙇

Reply:
Sadly not (that I'm aware of). I got the policy by manually creating a CloudTrail, encrypting it from the console, and seeing what policy AWS automatically attached to the key. I'll add a comment to that effect